src/App.tsx

```diff
@@ -1,8 +1,35 @@
-import React from 'react';
-
-export const App: React.FC = () => (
-  <div className="App">
-    <p>Nothing was pressed yet</p>
-    <p>The last pressed key is [Enter]</p>
-  </div>
-);
+import { Component } from 'react';
+
+type State = {
+  message: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    message: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleDocumentPressedKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleDocumentPressedKey);
+  }
+
+  handleDocumentPressedKey = (event: KeyboardEvent) => {
+    if (event.key) {
+      this.setState({ message: `The last pressed key is ${event.key}` });
+    }
```

Comment:
Здесь можно было добавить квадратные скобки, чтобы печаталось как в изначальном примере :)
```suggestion
    if (event.key) {
      this.setState({ message: `The last pressed key is [${event.key}`] });
    }
```